be/src/column/object_column.h

```diff
@@ -122,6 +123,8 @@ class ObjectColumn : public ColumnFactory<Column, ObjectColumn<T>> {
 
     const uint8_t* deserialize_and_append(const uint8_t* pos) override;
 
+    bool deserialize_and_append(const Slice& src);
```

Comment:
JsonColumn is extend ObjectColumn, need implement the method?

Reply:
jsonColumn already have old deserialize_and_append implemented and follows the old convention, so no need to implement this.